bin/tacl_cl_parser.py

```diff
@@ -0,0 +1,353 @@
+
+    assert int(year_)
+
+    title_text = issue_info
+    title = etree.Element('title')
+    title.text = title_text
+    node.append(title)
+
+    if not is_tacl:
+        month_text = issue_info.split()[-2]  # blah blah blah month year
+        assert month_text in {'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'}
+        month = etree.Element('month')
+        month.text = month_text
+        node.append(month)
+
+    year = etree.Element('year')
+    year.text = year_
+    node.append(year)
+
+    return node
+
+if __name__ == '__main__':
+    import sys
+
+    if sys.version_info < (3,6):
+        sys.stderr.write("Python >=3.6 required.\n")
+        sys.exit(1)
+
+    args = parse_args()
+    logging.basicConfig(level=args.verbose)
+
+    is_tacl = "tacl" in args.year_root.stem
+
+    prefix = TACL if is_tacl else CL  # J for CL, Q for TACL.
+    year = args.year_root.stem.split(".")[1]
+    year_suffix = year[-2:]  # Feels hacky, too.
+    volume_id = prefix + year_suffix
+
+    volume = etree.Element('volume')
+    volume.attrib['id'] = volume_id
+
+    tacl_glob = "tacl.20*.*/tacl.20*.*.xml"
+    # volume_info = get_volume_info(list(args.year_root.glob("*.*.*/*.*.*.xml"))[0])
+    # volume.append(volume_info)
+
+    if args.pdf_save_destination:
+        save_destination = args.pdf_save_destination
+        write_to_here = save_destination / "pdf" / prefix / volume_id
+        write_to_here.mkdir(parents=True, exist_ok=True)  # destination / Q / Q18
+
+    if args.old_version:
+        old_version = etree.parse(args.old_version)
+        old_root = old_version.getroot()
+
+    previous_issue_info, issue_count = None, 0
+
+    i = 1  # Stupid non-enumerate counter because of "Erratum: " papers interleaved with real ones.
+    for xml in sorted(args.year_root.glob("*_a_*/*.xml")):
+        # print(xml)
+
+        papernode, issue_info = process_xml(xml, is_tacl)
+        if papernode is None or papernode.find('title').text.startswith("Erratum: “"):
+            continue
+
+
+        if issue_info != previous_issue_info:
+            # Emit the new volume info before the paper.
+            log.info(f"New issue; will number it {issue_count + 1}")
+            log.info(f"{issue_info} vs. {previous_issue_info}")
+            previous_issue_info, issue_count = issue_info, issue_count + 1
+            volume.append(issue_info_to_node(issue_info, year, volume_id, issue_count, is_tacl))
+        paper_id = papernode.attrib['id'] = get_paperid(xml, i, issue_count)
+
+
+        pdf = xml.with_suffix(".pdf")
+        if not pdf.is_file():
+            log.error("Missing pdf for " + xml.name)
+        elif args.pdf_save_destination:
+            destination = write_to_here / "{}-{}.pdf".format(volume_id, paper_id)
+            shutil.copyfile(pdf, destination)
+
+        url_text = "http://www.aclweb.org/anthology/{}-{}".format(volume_id, paper_id)
```

Comment:
Please use `https` to be consistent with the rest of the Anthology. Maybe also move this to a variable at the top?

Reply:
Done; committed.